components/BaseController.php

```diff
@@ -225,4 +230,58 @@ protected function returnSuccess($message = 'Request successful', $statusCode =
         Yii::$app->response->statusCode = $statusCode;
         return array_merge(['code' => $statusCode, 'message' => $message], $additional);
     }
+
+
+    /**
+     * Attach files to Content
+     *
+     * @param Content|null $content
+     * @return array
+     */
+    protected function attachFilesToContent(?Content $content): array
```

Comment:
Wouldn't that method something for the `ContentController` or `BaseContentController`?

Reply:
@luke- Currently the method id used from 2 places:

- `class ContentController extends BaseController`
- `class PostController extends BaseContentController`

it is why I placed it in the class `BaseController`, if you really want to remove the base method `BaseContentController::actionAttachFiles($id)` then I will move the method in the `ContentController` only.